src/components/ErrorBoundary/ErrorBoundary.test.js

```diff
@@ -1,16 +1,14 @@
 import React from "react";
-import ErrorBoundary from "./ErrorBoundary";
-
 import { shallow } from "enzyme";
 
+import ErrorBoundary from "./ErrorBoundary";
+
 describe("Error Boundary", () => {
-  it("generates a error message when an error is caught", () => {
+  it("renders without crashing and matches snapshot", () => {
     const wrapper = shallow(<ErrorBoundary />);
     wrapper.setState({
       hasError: true
     });
-    expect(wrapper.text()).toEqual(
-      "Error: Something has gone wrong. If this issue persists, please raise an issue on GitHub."
-    );
+    expect(wrapper).toMatchSnapshot();
```

Comment:
For this to work properly, you may have to put it in the callback for `setState` and then make this test async by adding `done`. See comments below about the snapshot not working.

Reply:
This test would be better if it simulated an error condition for an error boundary, rather than asserting that setting the internal state does what you expect.

You can use `wrapper.find(ChildComponent).simulateError(error);` to simulate a child component throwing an error, and then check that the ErrorBoundary renders appropriately.

Reply:
Ignore my comment about the `setState` stuff, I was going down the wrong rabbit hole.

@barrymcgee I believe that we discussed what @squidsoup is proposing. He has said it much more eloquently and should probably be done that way 😺

Reply:
done - throwing an error means Jest test console spits out the stack trace every time though. Could get noisy over time? Jest has a `verbose` config setting but it's not supported by CRA overrides. 
<img width="1120" alt="Screenshot 2019-08-13 at 12 07 28" src="https://user-images.githubusercontent.com/505570/62936959-0d3ec280-bdc3-11e9-9227-99476f8d185e.png">